freeplane/src/main/java/org/freeplane/view/swing/map/overview/MapOverviewUtils.java

```diff
@@ -0,0 +1,100 @@
+package org.freeplane.view.swing.map.overview;
+
+import java.awt.Color;
+import java.awt.Insets;
+import java.awt.Rectangle;
+
+import javax.swing.JScrollBar;
+import javax.swing.JScrollPane;
+
+import org.freeplane.core.resources.ResourceController;
+import org.freeplane.view.swing.map.MapView;
+
+public class MapOverviewUtils {
```

Comment:
In recent years, I become more affined to clean code and sensitive to ways to improve it, so I hope you allow me to share some ideas with you as I see ways to improve the code.

Having a collection of static methods inside a utility class always seems suspicious. Actually, such classes are needed only in a specific situation when you want to extend some existing library or framework in a generic way. And even in such cases it is better to create a separate extention class per extended class or even per extension. 

Actually even a word Utilities (or its abbreviation)  as a part of the class name is suspicious.